ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/resources/KsqlResourceTest.java

```diff
@@ -375,7 +375,7 @@ public void shouldListFunctions() {
 
     // not going to check every function
     assertThat(functionList.getFunctions(), hasItems(
-        new SimpleFunctionInfo("TIMESTAMPTOSTRING", FunctionType.scalar),
+        new SimpleFunctionInfo("EXTRACTJSONFIELD", FunctionType.scalar),
```

Comment:
Functions that use the new Udf annotation don't show up in this list. Changing the test to use another Kudf based function.